app/src/pages/home/homeLoader.ts

```diff
@@ -33,6 +34,11 @@ export async function homeLoader(_args: LoaderFunctionArgs) {
   if (data?.functionality.modelInferences) {
     return redirect("/model");
   } else if (data?.projects.edges?.length) {
+    for (const { project } of data.projects.edges) {
+      if (project.endTime != null) {
+        return redirect(`/projects/${project.id}`);
+      }
+    }
```

Comment:
this is going to fail when all projects are empty. We need to fall back to the default one at least.

Reply:
that's covered by line 42 because default is always at the first position